### PR TITLE
Add largest keyword distance assessment to assessor based on assessor options

### LINKED ISSUE #1566: Add keywordDistribution to assessor in premium
We should add functionality that adds the keyword distribution assessment to the SEO assessor in Premium. This assessment should not be triggered in Free.

### PR DESCRIPTION
**Pull request by manuelaugustin**
## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

* Adds functionality that makes it possible to add the largest keyoword distance assessment based on assessor options.

## Test instructions

This PR can be tested by following these steps:

* Make a beta script combining this branch with `stories/1566-add-keywordDistribution-to-SEOAssessor-in-premium` in Yoast SEO Premium (see https://github.com/Yoast/wordpress-seo-premium/pull/1819).
* Add a long text (>200 words) and set a keyword. Add the keyword at the beginning and the end of the text. Make sure you get feedback about your keyword distribution.

Fixes #1566

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -315,36 +341,35 @@ App.prototype.getDefaultOutputElement = function( args ) {
  * @returns {void}
  */
 App.prototype.switchAssessors = function( useCornerStone ) {
```
Comment (Xyfi) [inline src/app.js]: If I understand correctly this function sets the app's assessors based on the `._assessorOptions`, just like `changeAssessorOptions` does but this one is specifically for `userCornerStone` only. I guess this is due to backwards compatibility, but can't we think of a cleaner way to solve this?

Comment (manuelaugustin) [reply src/app.js]: As discussed, I added `@deprecated` to the documentation of switchAssessors.

**Comment by manuelaugustin** [comment]
CR of the last commit done 👍

**Comment by Xyfi** [comment]
Acceptance :+1:

**Pull request merged**